esp/services/ws_ecl/ws_ecl_service.hpp

```diff
@@ -140,6 +140,12 @@ class CWsEclBinding : public CHttpSoapBinding
 private:
     CWsEclService *wsecl;
 
+protected:
+    bool getSchema(StringBuffer& schema, IEspContext &ctx, CHttpRequest* req, const char *service, const char *method,bool standalone) override;
```

Comment:
Was there any specific reason this was moved to being protected from public?

Reply:
This function with the final bool parameter wasn't overridden in this class before, but it is/was protected in the base class EspHttpBinding- I think it just looks confusing because I renamed & moved the other one.

The other `getSchema` function (with the final  wuinfo parameter) I moved from public to protected in this class and renamed to `getSimpleSchema` (on your suggestion- seemed reasonable especially now that it makes sense to refactor it away in favor of the other one).